src/index.tsx

```diff
@@ -35,6 +40,18 @@ history.listen(trackPageView);
 
 const store = createStore({});
 
+// check for existing user stats in persisted redux store 
+if (store.getState().userStatsState.all.length === 0) {
+  // check for user stats from legacy choo app
+  const legacyStatsStore = getAll('org.5calls.userStats');
+
+  if (legacyStatsStore && Array.isArray(legacyStatsStore) && legacyStatsStore[0]) {
+    const legacyStats: LegacyUserStatsState = legacyStatsStore[0];
+    let transformedLegacyStats = transform(legacyStats);
+    store.dispatch(setUserStatsActionCreator(transformedLegacyStats));
+  }
```

Comment:
I would think that `org.5calls.userStats` needs to be removed from local storage here so that the same stats do not get re-migrated everytime a user visits the app.

Reply:
It only migrates when there's no new stats (`userStatsState.all.length === 0`) so there's no harm in leaving it for now. A bit of caution if we need to roll back to choo for some reason.

Reply:
OK. Then we need to delete the `localStorage.removeItem()` call from `redux/remoteData/asyncActionCreator.migrateLegacyCompletedIssues()`

Reply:
Thanks for reminding me, I had the line to remove the choo legacy stats from local storage, but hadn't added it while I was testing.  It would just be calling the "remove" method from the "LocalStorage" module after reading the stats in to src/redux in index.tsx.  We can wait to add it until you're ready.